pkg/event-manager/subscriptions/manager.go

```diff
@@ -153,3 +159,18 @@ func (m *defaultManager) runFunction(fnName string, event *events.CloudEvent, se
 	log.Debugf("Function %s returned %+v", result.FunctionName, result.Output)
 	return
 }
+
+func (m *defaultManager) processEventData(event *events.CloudEvent) (interface{}, error) {
+	switch event.ContentType {
+	case "application/json":
```

Comment:
To be consistent with #399, should this also support "application/cloudevents+json"?

Reply:
In my opinion no. My understanding is that `application/cloudevents+json` represents the entire CloudEvent envelope (with all metadata). `processEventData` only considers the content type of the `data` attribute of the event.

(There is the edge case of "hey look! I nested event inside the event!", but that falls into supporting more content types in general).

Reply:
I believe the edge case is: "Yo dawg, I heard you like CloudEvents, so we put a CloudEvent in your CloudEvent so you can trigger while you trigger!"